config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -136,3 +136,19 @@ spec:
     - key: codecovToken
       name: codecov-token
       version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
```

Comment:
What is this secret?
Can you please work with SIG K8s Infra to provide resources through the CNCF?

Note that while the legacy Google internally run cluster has provided support for external secrets to unblock non-googlers setting things up, the community clusters have NOT supported this and only have community provisioned secrets through community funding sources.